VariablesHomework/VariablesHomework.cs

```diff
@@ -161,18 +174,18 @@ public void Casting()
             // what am i doing here that make the above work?
             double d = 15.5;
             int c = (int)d;
-            Assert.IsTrue(c == /*Answer here*/); // Make this pass
+            Assert.IsTrue(c == 15/*Answer here*/); // Make this pass
 
             string e = d.ToString();
-            Assert.IsTrue(e == /*Answer here*/); // Make this pass
+            Assert.IsTrue(e == 15.ToString()/*Answer here*/); // Make this pass
```

Comment:
We haven't looked at conversions yet, but this isn't what I'm looking for here. It should be just a literal value.